owasp-top10-2016-mobile/m2/cool_games/README.md

```diff
@@ -0,0 +1,178 @@
+
+To start this intentionally **insecure application**, you will need [Flutter][VS-Code-Flutter], [Docker][Docker Install] and [Docker Compose][Docker Compose Install]. After forking [secDevLabs](https://github.com/globocom/secDevLabs), you'll need to start the server, which can be done through the commands:
+
+### Start server commands:
+
+```sh
+cd secDevLabs/owasp-top10-2016-mobile/m2/cool_games/server
+```
+
+```sh
+make install
+```
+
+### Start app commands:
+
+**Note**: It is important to mention, that you should have an emulator up and running or an available device to run the app in. For instructions on how to set up an emulator, click [here][VS-Code-Flutter].
+
+After properly setting up your editor and emulator/device to run Flutter apps, open the `main.dart` file with VS Code. You can find it's path here:
+
+```
+secDevLabs/owasp-top10-2016-mobile/m2/cool_games/mobile/lib/main.dart
+```
+
+Now, all you need to do is to click `Run > Run Without Debugging` on VS Code top menu bar and it should begin building the app to launch it in no time!
+
+Then, you should see Cool Games' app launch successfully in the emulator/device you're using! 📲
+
+**Note**: In case an Android licenses errors shows up, you may need to run the following command on your terminal and accept them:
+
+```sh
+flutter doctor --android-licenses
+```
+
+## Get to know the app 📝
+
+To properly understand how this application works, you can follow these simple steps:
+
+- Register a new user. 👩‍💻
+- Get yourself up to date with the latest gaming trends! 🎮
+
+## Attack narrative
+
+Now that you know the purpose of this app, what could go wrong? The following section describes how an attacker could identify and eventually exploit some of the app's flaws to compromise it's users. We encourage you to follow these steps and try to reproduce them on your own to better understand the attack vector! 😜
+
+### 👀
+
+#### Logs generated on log in with user credentials allows for possible leak of sensitive information
+
+When first accessing the application, an user is required to authenticate to access the gaming content, as we can see from the image below:
+
+<p align="center">
+    <img src="images/log_in.png"/>
+</p>
+
+At first hand, the login flow seemed to have run flawlessly, as the user can now access the gaming news, as shown below:
+
+<p align="center">
+    <img src="images/successfull_log_in.gif"/>
+</p>
+
+### 🔥
+
+If an attacker came into posession of the device and used the [Android Debug Bridge](https://developer.android.com/studio/command-line/adb) tool to communicate with it through a Unix shell, he could inspect how the app behaves and what it logs.
+
+To begin, after connecting the device through an USB port, it is possible to list connected devices with the following command:
```

Comment:
Since we're using a virtual device this instruction to `connect the device through an USB port` may be misleading/confusing.

Reply:
You're right, @joserenatosilva! I'll remove that!